h/models/__init__.py

```diff
@@ -51,6 +52,7 @@
     'FeatureCohort',
     'Flag',
     'Group',
+    'Organization',
```

Comment:
Can we keep these in alphabetical order please (ie. Organization below GroupScope).